VAMobile/src/utils/jsonFormatting.ts

```diff
@@ -17,3 +17,10 @@ export const fixedWhiteSpaceString = (text: string | undefined) => {
   }
   return ''
 }
+
+export const fixSpecialCharacters = (text: string): string => {
```

Comment:
A more specific name would be `translateHTMLEntities`. Up to you if you want to change it or not

Reply:
I think we are good with this name, it allows us to fix things that might not be html related in the same function if needed in the future